api/libproject/devstart.rb

```diff
@@ -1609,6 +1639,11 @@ def get_auth_domain(project)
   return JSON.parse(File.read("config/#{config_json}"))["firecloud"]["registeredDomainName"]
 end
 
+def get_es_base_url(env)
```

Comment:
Side note: I didn't fully realize that our config_*.json files were also being used to control deploy-side scripts. I guess it's attractive to centralize the source of truth. But it feels a bit awkward (and possibly risky / difficult to reason about) that we're using the config to control both the runtime environment and some characteristics of how that environment gets deployed.

For instance: I would definitely feel less safe about making an emergency config push directly to the SQL database, given that the deploy script is assuming some consistency between the deploy logic and the pushed config.

I can't quite decide if this is this an actual risk, or just a theoretical / hypothetical. Food for thought...

Reply:
There's only a handful of commands here that do this. One of them is the command to `load_config` (which is actually the command that pushes the config). The rest of these are unrelated to deployment.

Without some higher order config language I'm not sure we can avoid this without duplicating flag values in various places.